nondex-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'groovy'
+apply plugin: 'maven'
+
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+
+    }
+    dependencies {
+        classpath "com.gradle.publish:plugin-publish-plugin:0.9.7"
+    }
+}
+apply plugin: "com.gradle.plugin-publish"
+repositories {
+    jcenter()
+    mavenLocal()
+}
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+    compile 'edu.illinois:nondex-common:1.1.2-SNAPSHOT'
+    compile 'edu.illinois:nondex-instrumentation:1.1.2-SNAPSHOT'
+    testCompile 'junit:junit:4.12'
+}
+
+version = "1.2.1"
```

Comment:
This version number is for the gradle plugin. I think she has released some versions of this plugin before so the number is now "1.2.1". Should I change this to be consistent with the maven plugin version number?